CONTRIBUTING.md

```diff
@@ -33,6 +33,8 @@ $ yarn new source my-new-adapter
 
 _If on a Mac, this requires `gnu-sed` to be installed and set as the default for the command `sed`._
 
+Follow the checklist outlined in [New EA PR Template](./.github/PULL_REQUEST_TEMPLATE/new_ea_pr_template.md) for the remaining development steps.
```

Comment:
How about including an image of how to select the PR template?

Reply:
Apparently there isn't a UI to select the PR template like there is for issues, but I can add a line about using URL params like this: `https://github.com/smartcontractkit/external-adapters-js/compare/develop...<branch>?quick_pull=1`